### ISSUE TITLE
Subsequent use of bold fails

### ISSUE THREAD

**Issue by Simon-Hohberg** [showdownjs/showdown#486]
Showdown __is__ a __Javascript__ Markdown to HTML converter ...

produces

Showdown __is____ __a__ ____Javascript__ Markdown to HTML converter ...

**Comment by tivie** [comment]
What options have you enabled?

**Comment by tivie** [comment]
I assume you're using literalmidwordsunderscore option.

Seems this bug was introduced in version 1.6.4

**Comment by Simon-Hohberg** [comment]
Yes, you are right, it happens only with `literalmidwordsunderscore` activated.

**Comment by tivie** [comment]
It's a bug nonetheless
